Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```
@@ -155,6 +155,23 @@
   - type: Fiber
     fiberMaterial: fibers-nitrile
   - type: FingerprintMask
+
+- type: entity
+  parent: ClothingHandsBase
+  id: ClothingHandsGlovesCombatNitrile
+  name: nitrile combat gloves
+  description: Very hight-quality nitrile combat gloves. Designed to provide medical assistance during hostilities.
+  components:
+  - type: Insulated
+  - type: Sprite
+    sprite: Clothing/Hands/Gloves/nitrile.rsi
+  - type: Clothing
+    sprite: Clothing/Hands/Gloves/nitrile.rsi
+  - type: Fiber
+    fiberMaterial: insulative
```

```suggestion
    fiberMaterial: fibers-insulative
```
You got this word wrong, which is why the tests aren't happy